pkgs/desktops/gnome/default.nix

```diff
@@ -355,7 +359,9 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   maintainers = lib.teams.gnome.members;
 
-  mutter328 = throw "Removed as Pantheon is upgraded to mutter334.";
+  mutter328 = throw "Removed as Pantheon is upgraded to mutter338.";
+
+  mutter334 = throw "Removed as Pantheon is upgraded to mutter338.";
 
```

Comment:
Can you move those to aliases.nix if possible?

Reply:
There are other aliases in this file, do you mean create a new `pkgs/desktops/gnome/aliases.nix` and move all of the aliases to the file or something else? 🤔

Reply:
No, just move them all in one section to aliases.nix. We only have extra files for other package sets because they are in other ways special. I don't think that applies here.

Reply:
Top-level aliases.nix does not support creating aliases inside existing scopes. (At least not without replacing it with a new overriden scope.)